src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/EvaluationContext.cs

```diff
@@ -183,6 +187,7 @@ internal sealed class EvaluationContext : EvaluationContextBase
             var localSignatureHandle = (localSignatureToken != 0) ? (StandaloneSignatureHandle)MetadataTokens.Handle(localSignatureToken) : default(StandaloneSignatureHandle);
 
             var currentFrame = compilation.GetMethod(moduleVersionId, methodHandle);
+            var currentSourceMethod = compilation.GetSourceMethod(moduleVersionId, methodToken);
```

Comment:
I like the idea. Thanks!

---
In reply to: [134928456](https://github.com/dotnet/roslyn/pull/21707#discussion_r134928456) [](ancestors = 134928456)